### PR TITLE
fixed minor issue in no_short_password

### PR DESCRIPTION
**Pull request by manjunath24**
fixed minor issue in no_short_password

### CODE REVIEW

File: secure_login/checkers.py [hunk 1]
```diff
@@ -16,10 +16,11 @@ def no_weak_passwords(username, password, **kwargs):
 
 
 def no_short_passwords(username, password, **kwargs):
-    if (len(password) <
-            getattr(settings, "SECURE_LOGIN_MIN_PASSWORD_LENGTH", 6)):
-        return False
-    return True
+    if password:
+        if (len(password) <
```
Comment (shabda) [inline secure_login/checkers.py]: @manjunath24 : 
Its simpler to do this as `if password and len(password)` rather than a nested if.

Comment (manjunath24) [inline secure_login/checkers.py]: @shabda  :
Thanks shabda, will correct this.

**Comment by shabda** [comment]
@manjunath24 : pep8 failed in travis. Also can you add a test for this fix?

**Comment by manjunath24** [comment]
Okay @shabda will add test for these changes.

**Comment by manjunath24** [comment]
@shabda  I didn't find lint file to check pep8.Do i need to create one?

**Comment by shabda** [comment]
@manjunath24 : We are only running this right now 
`pep8 --ignore=E501 secure_login/`. See travis.yml

**Pull request merged**